recipes/sp2/meta.yaml

```
@@ -0,0 +1,93 @@
+{% set name = "sp2" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aeddd557b706de7d72c1952a0d6da0348b4d6f6001b8ef643dec915e8eac1d66
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - export SP2_USE_VCPKG=0  # [not win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps  # [not win]
+    - set PSP_GENERATOR=Visual Studio 16 2019  # [win]
+    - set SP2_USE_VCPKG=0  # [win]
+    - {{ PYTHON }} setup.py build_ext install --single-version-externally-managed  --record=record.txt  # [win]
+  rpaths:
+    - {{ SP_DIR }}/sp2/lib
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+  host:
+    - python
+    - bison
+    - brotli
+    - cmake
+    - cyrus-sasl
+    - exprtk
+    - flex
+    - gtest
+    - libarrow =15
+    - libparquet =15
+    - libprotobuf
+    - librdkafka
+    - lz4-c
+    - numpy
+    - pip
+    - pyarrow =15
+    - rapidjson
+    - ruamel.yaml
+    - scikit-build
+    - setuptools >=69
+    - tar
+    - unzip
+    - zip
+    - zlib
+  run:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+    - python
+    - {{ pin_compatible('numpy') }}
+    - brotli
+    - cyrus-sasl
+    - libarrow =15
+    - libprotobuf
+    - librdkafka
+    - lz4-c
+    - packaging
+    - pandas <2.2
+    - pyarrow =15
+    - psutil
+    - pytz
+    - rapidjson
+    - ruamel.yaml
+    - sqlalchemy
+    - zlib
```

Most of these come in automatically via `run_exports`. Please then also let `run_exports` do its thing as some of them only install a subset of the development package at runtime.
```suggestion
    - packaging
    - pandas <2.2
    - pyarrow =15
    - psutil
    - pytz
    - rapidjson
    - ruamel.yaml
    - sqlalchemy
```